components/lib/VDropdown.vue

```diff
@@ -58,6 +58,11 @@ export default {
       isOpen: false,
     }
   },
+  mounted() {
+    document.addEventListener('click', () => {
+      this.isOpen = false
+    })
+  },
```

Comment:
Added this code to close the expanded dropdown panel if clicked on document

Reply:
We'll also need to remove this event listener before the component is unmounted.

```
methods: {
  closeOnOutsideClick() {
    this.isOpen = false
  }
},
mounted() {
  document.addEventListener('click', this.closeOnOutsideClick)
},
beforeDestroy() {
  document.removeEventListener('click', this.closeOnOutsideClick)
}
```

Reply:
@punaifang updated this